media/com_plugins/js/edit.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+Joomla.submitbutton = (task) => {
+  if (task === 'plugin.cancel' || document.formvalidator.isValid(document.getElementById('style-form'))) {
+    Joomla.submitform(task, document.getElementById('style-form'));
+  }
+
+  if (task !== 'plugin.apply') {
+    if (self !== top) {
```

Comment:
Do you know where this is used?

Reply:
or anybody else? @dgt41 ?

Reply:
@dneukirchen To be honest, I do not know. That is why I asked. I did not know where to test it.

Reply:
Well... if nobody (ping @dgt41 @C-Lodder @Fedik ) knows where this is actually used, we should remove the script.

Reply:
no idea, need to search

Reply:
I cant find any usage. So i guess wie can remove this inline script completely.